File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest05262.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest05262 {

    @GetMapping("/BenchmarkTest05262")
    public void BenchmarkTest05262(@RequestHeader("Origin") String origin, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String originValue = origin != null ? origin : "";
        String data = java.util.concurrent.CompletableFuture
            .supplyAsync(() -> originValue)
            .thenApply(v -> v.strip().replaceAll("\\s+", " "))
            .join();
        String processed = data.length() > 64 ? data.substring(0, 64) : data;
        response.getWriter().print("{\"resource\":\"" + processed + "\"}");
    }
}

File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_05262.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify
from app_runtime import auth_check


def BenchmarkTest05262():
    user_id = request.args.get('id', '')
    parts = str(user_id).split(',')
    data = ','.join(parts)
    auth_check('user', data)
    return jsonify({"result": "success"})
